Project.toml

```diff
@@ -9,21 +9,23 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
 PyPlot = "d330b81b-6aea-500a-939a-2ce795aea3ee"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StructTypes = "856f2bd8-1eba-4b0a-8007-ebc267875bd4"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-LightGraphs = "^1.1.0"
-StatsBase = "0.32"
-DataStructures = "0.17.7"
 Conda = "1.3.0"
+DataStructures = "0.17.7"
 JSON = "0.21.0"
 JSON3 = "0.1.13"
+LightGraphs = "^1.1.0"
```

Comment:
I think we should be stricter and allow just `"1.1"`

Reply:
The current version is `1.3.3` so let's set it to `1.3` (there are major performance differences between 1.1 and 1.3 and tests pass on 1.3 anyway)

Reply:
I did not modify the current LightGraph version

Reply:
What I say that `"^1.1.0"` is not a good dependency specifier. Packages should not use `^` in dependency specifications.

Reply:
Bogumil is suggesting to pin it to a single version. "^1.1.0" means any "1.*.*" version (that is any realeas having number lower than 2.0.0 ). "1.3" will allow any 1.3.*  but will not allow 1.4.0 (a package update will be needed at that point. So the Bogumil's suggestion is that you update it to "1.3" within this PR. And yes you have not been changing that :-)

Reply:
@bkamins just to make sure, according to https://docs.julialang.org/en/v1.0/stdlib/Pkg/
```
1.2.3 == ^1.2.3
```
So I assume that you mean that the `^` character is obsolete (not needed) or do you suggest something else?

Reply:
@pszufe many thanks for your explanation, now it is clearer. My previous comment was incomplete: I did not modify the LightGraph version when fixing this PR as I was not sure which one I had to use :)

Reply:
@bkamins looking one more time on documentation:
```
1.3 == ^1.3 == [1.3.0, 2.0.0)
1.3.1 == ^1.3.1 == [1.3.1, 2.0.0)
```
So I am not sure about your comment why 1.3 is "stricter". Julia package specification is tricky so perhaps I missed something here?

Reply:
perhaps you meant `~1.3`  which is equivalent to `[1.3.0, 1.4.0)`?

Reply:
I exactly mean that `^` is obsolete and just commenting on the practice that is typically followed by other packages (but this is a mild issue).

Reply:
@bkamins  So we do here:
```
LightGraphs = "1.3"
```
OK?

Reply:
exactly "simple is best" :). Thank you!